### PR TITLE
feat: DIA-1325: update response for inference-runs

### PR DESCRIPTION
**Pull request by pakelley**
- Updated functions:
  ```
  ls.prompts.runs.list()
  ls.prompts.runs.create()
  ```
- Key Indicators API:
 ```python
 key_indicators = ls.prompts.indicators.list(run_id=123)
 for ki in key_indicators:
    value = ls.prompts.indicators(run_id=123, key=ki.key)
 ```

### CODE REVIEW

File: fern/openapi/overrides.yaml [hunk 1]
```diff
@@ -24,15 +24,43 @@ paths:
...
       x-fern-sdk-group-name:
         - prompts
-        - versions
-      x-fern-sdk-method-name: create_run
+        - runs
+      x-fern-sdk-method-name: create
+      x-fern-audiences:
+        - public
+
+  /api/inference-runs/{pk}/indicators:
```
Comment (matt-bernstein) [inline fern/openapi/overrides.yaml:44]: can this get a `list()` method to list all valid indicator names for this inference run? Seems like something the FE would need, and if not, still good for discoverability in the SDK

Comment (pakelley) [reply fern/openapi/overrides.yaml:44]: This method itself is basically the list function. It does exactly that: tells the FE which metrics to query using the {indicator_key} based api just below

Comment (matt-bernstein) [reply fern/openapi/overrides.yaml:44]: How would you do that in the SDK?

Comment (matt-bernstein) [reply fern/openapi/overrides.yaml:44]: I see, nvm

**Pull request merged**